src/app/home/form/form.component.ts

```diff
@@ -0,0 +1,24 @@
+// import { FormService } from './form.service';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.css'],
+})
+export class FormComponent {
+  // private formService: FormService;
+  formClass: string = 'container';
+  
+
+ 
+  //toggling container class
+
+  
```

Comment:
try to remove unecessary new lines and spaces at different places

Reply:
ok sir

Reply:
lol